### ISSUE TITLE
VPN Window does not display any content on Linux Lubundu (LXDE interface)

### ISSUE THREAD

**Issue by JKingZapata** [mozilla-mobile/mozilla-vpn-client#1462]
**Description**
VPN installed on Lubundu (LXDE interface) based on Ubuntu LTS version (Groovy Gorilla 20.10). but when the Mozilla VPN window opens but does not display any content.

**Technical Information**

* OS: Lubundu (LXDE interface)
* VPN version: 2.3.3 

**Steps to Reproduce**

1. Pay Mozilla VPN subscription
1. run Linux install commands from terminal
1. Execute mozillavpn
1. VPN screen is blank and does not display any options to connect

Attaching debug logs
[mozillavpn-2021-7-21.txt](https://github.com/mozilla-mobile/mozilla-vpn-client/files/6902992/mozillavpn-2021-7-21.txt)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-834)

**Comment by oskirby** [comment]
I'm not having any luck reproducing this issue, so I'm beginning to wonder if there's something missing in either the bug description or my test setup.

I've noticed that the log file doesn't quite match the description. Lubuntu LTS would be version 20.04, while groovy gorilla is 20.10, and the log file shows a user agent of ubuntu 20.04. I've tried installing the VPN client on both Lubuntu 20.04 and 20.10 virtual machines but neither of which seem to reproduce the issue.
